docs/usage/pages.rst

```diff
@@ -72,23 +98,24 @@ extra fields. For example:
         def foo(self):
             return "bar"
 
-.. _extractors:
-
-Extractors
-==========
+Fields of these classes have ``auto_field`` set to ``True`` in their field
+metadata, so that you can check if a page object subclass is overriding a field
+using :func:`~zyte_common_items.fields.is_auto_field`:
 
-For some nested fields (:class:`~.ProductFromList`, :class:`~.ProductVariant`),
-:ref:`base extractors <default-processors-nested>` exist that you can subclass
-to write your own extractors.
+.. autofunction:: zyte_common_items.fields.is_auto_field
 
-They provide the following base line:
-
--   They declare the :ref:`item class <items>` that they return, allowing for
-    their ``to_item`` method to automatically build an instance of it from
-    ``@field``-decorated methods. See :ref:`fields`.
+.. code-block:: python
 
--   They also provide default :ref:`processors <processors>` for some
-    item-specific fields.
+    print(is_auto_field(ExtendedProductPage, "name"))  # Returns False
+    print(is_auto_field(ExtendedProductPage, "foo"))  # Returns False
+    print(is_auto_field(ExtendedProductPage, "brand"))  # Returns True
+    print(is_auto_field(ExtendedProductPage, "bar"))  # Raises KeyError
 
-See :ref:`extractor-api`.
+If you are overriding a field method but the method continues to return the
+value straight from the ``Auto``-prefixed class, e.g. because you are only
+overriding field metadata or :ref:`processors <processors>`, you should also
+set ``auto_field`` to ``True``. Instead of setting it manually in the field
```

Comment:
I think it depends on how we want to use it. For example, if an item value comes from Zyte API, but a processor is needed, I can imagine the value was not right. In this case, an override is needed, and probably it's better to indicate it by not using auto_field.

Reply:
I'm not 100% sure what are the scenarios where you do want to mark it as auto (maybe some stats, logging, etc?).

Reply:
We also have an option to not document the scenarios for now.